shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -329,12 +329,16 @@ - (void)touchesCancelled:(NSSet*)touches withEvent:(UIEvent*)event {
 #pragma mark - Handle view resizing
 
 - (void)updateViewportMetrics {
+  // Update viewport metrics for the engine.
   blink::Threads::UI()->PostTask([
     engine = _platformView->engine().GetWeakPtr(), metrics = _viewportMetrics
   ] {
     if (engine.get())
       engine->SetViewportMetrics(metrics);
   });
+
+  // Tell the platform view that its surface was updated.
+  _platformView->SurfaceSizeChanged();
```

Comment:
The backing store is only updated if the surface size is actually changed (what eventually boils down to `UpdateStorageSizeIfNecessary`). So the overhead of having this here is only limited to the thread hopping in case there is no actually size changes. I am not too sure if there are edge cases where the surface size actually changes when there is the double padding introduced, etc.. Just thought it was be safer this way. Can add the check if you wish.